recipe/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+if [[ `uname` == 'Darwin' ]]; then
+	CXXFLAGS="${CXXFLAGS} -DBOOST_NO_CXX11_RVALUE_REFERENCES"
+	MACOSX_DEPLOYMENT_TARGET=10.9
+	$PYTHON -B setup.py install --single-version-externally-managed --record record.txt
```

Comment:
@tiagofrepereira2012 could you please take `$PYTHON -B setup.py install --single-version-externally-managed --record record.txt` line out of the `if` statement.